english/Pam_and_Jim

```diff
@@ -0,0 +1 @@
+Pam and Jim are the greatest couple in the history of tv!
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.